client/src/components/menus/AddChatView.tsx

```diff
@@ -32,7 +32,10 @@ const AddChatView = () => {
   };
 
   const handleCreateChat = async () => {
-    await createChat(chatName, addedUsers, profilePicture);
+    const response = await createChat(chatName, addedUsers, profilePicture);
+    if (response) {
```

Comment:
to zawsze będzie truthy, response może być pustym obiektem a i tak zabangla.

trzeba sprawdzić utilsem `checkObjectEmpty` i jak false to wtedy to zrobić